config/settings.yml

```diff
@@ -0,0 +1,142 @@
+ps_reg_endpoint: 'http://registrar.example.edu'
+ps_reg_username: ''
+ps_reg_password: ''
+# workspace_dir: <%= Rails.root.join('public', 'workspace').to_s %>
+# preview_dir: <%= Rails.root.join('tmp', 'preview').to_s %>
```

Comment:
I left these 2 commented. I suspect we do not need them, but for now wanted to keep them as indicators of something to look out for when porting code. I suspect there is a good bit of clean up in this file that can be done before we go live with this repo.